examples/prom-grafana/lib/ksonnet-util/kausal.libsonnet

```diff
@@ -0,0 +1,432 @@
+// Override defaults paramters for objects in the ksonnet libs here.
```

Comment:
why is this file in `lib` not `vendor`?

Reply:
Because nobody reviewed https://github.com/grafana/jsonnet-libs/pull/192 yet, so I had to override the one in vendor